compiler/src/dotty/tools/dotc/core/TypeErasure.scala

```diff
@@ -95,6 +95,8 @@ object TypeErasure {
   abstract case class ErasedValueType(tycon: TypeRef, erasedUnderlying: Type)
   extends CachedGroundType with ValueType {
     override def computeHash(bs: Hashable.Binders): Int = doHash(bs, tycon, erasedUnderlying)
+
+    final def valueClassSymbol(using Context): ClassSymbol = tycon.typeSymbol.asClass
```

Comment:
Self-review: this is used only once in `JSCodeGen`. It's probably not worth adding.